components/automate-cli/cmd/chef-automate/ha_node_operations_utils.go

```diff
@@ -217,9 +217,11 @@ func (nu *NodeUtilsImpl) isA2HARBFileExist() bool {
 }
 
 func (nu *NodeUtilsImpl) taintTerraform(path string) error {
+	modeOfDeployment := getModeOfDeployment()
+	if modeOfDeployment == AWS_MODE {
 	if err := executeShellCommand("/bin/bash", []string{"-c", REMOVE_TERRAFORM_STATE}, filepath.Join(path, "destroy", "aws")); err != nil {
 		return err
-	}
+	}}
```

Comment:
indentation seems not correct, please use gofmt in your vscode plugin to run every time you save go file

Reply:
sure